### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by Mickss**

### CODE REVIEW

File: src/test/java/org/micks/champmaker/championships/ChampionshipServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowable;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class ChampionshipServiceTest {
+
```
Comment (kbnowak) [inline src/test/java/org/micks/champmaker/championships/ChampionshipServiceTest.java:18]: (n2h) You can delete first blank line of the class.

File: src/test/java/org/micks/champmaker/championships/ChampionshipServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    @InjectMocks
+    private ChampionshipService championshipService;
+
+    @Mock
+    private ChampionshipRepository championshipRepository;
+
+    @Test
+    void shouldReturnChampionship() throws EntityNotFoundException {
+        // given
+
```
Comment (kbnowak) [inline src/test/java/org/micks/champmaker/championships/ChampionshipServiceTest.java:27]: I'd remove blank lines after //given //when and //then

File: src/test/java/org/micks/champmaker/championships/ChampionshipServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        Optional<ChampionshipEntity> mockedChampionship = Optional.of(new ChampionshipEntity("Pomeranian", "Szczecin", "15 August"));
```
Comment (kbnowak) [inline src/test/java/org/micks/champmaker/championships/ChampionshipServiceTest.java]:
```suggestion
        Optional<ChampionshipEntity> expectedChampionship = Optional.of(new ChampionshipEntity("Pomeranian", "Szczecin", "15 August"));
```
This object is not mocked, it's real test object and you later do assertions against it's values.

File: src/test/java/org/micks/champmaker/championships/ChampionshipServiceTest.java [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+        when(championshipRepository.findById(322L)).thenReturn(mockedChampionship);
+
+        // when
+
+        ChampionshipDTO championship = championshipService.getChampionship(322);
+
+        // then
+
+        assertThat(championship.getCity()).isEqualTo("Szczecin");
```
Comment (kbnowak) [inline src/test/java/org/micks/champmaker/championships/ChampionshipServiceTest.java:36]: Assert all of the object's values, or just the whole championship object at once.
```suggestion
        assertThat(championship).isEqualTo(expectedChampionship);
```

File: src/test/java/org/micks/champmaker/championships/ChampionshipServiceTest.java [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+        Throwable throwable = catchThrowable(() -> championshipService.getChampionship(322));
+
+        // then
+
+        assertThat(throwable)
+                .isInstanceOf(EntityNotFoundException.class)
+                .hasMessage("Can not find Championship with Id: 322");
+    }
+}
```
Comment (kbnowak) [inline src/test/java/org/micks/champmaker/championships/ChampionshipServiceTest.java]: Add empty line at the end of each text file.

File: src/test/java/org/micks/champmaker/championships/ChampionshipServiceTest.java (hunk 1 above)
Comment (tknowak) [reply src/test/java/org/micks/champmaker/championships/ChampionshipServiceTest.java:18]: I like when field or method has empty line before 🙂

File: src/test/java/org/micks/champmaker/championships/ChampionshipServiceTest.java (hunk 2 above)
Comment (Mickss) [reply src/test/java/org/micks/champmaker/championships/ChampionshipServiceTest.java:27]: Fixed

File: src/test/java/org/micks/champmaker/championships/ChampionshipServiceTest.java (hunk 3 above)
Comment (Mickss) [reply src/test/java/org/micks/champmaker/championships/ChampionshipServiceTest.java]: Fixed

File: src/test/java/org/micks/champmaker/championships/ChampionshipServiceTest.java (hunk 4 above)
Comment (Mickss) [reply src/test/java/org/micks/champmaker/championships/ChampionshipServiceTest.java:36]: Added two more assertions. It is more readable for me :)

File: src/test/java/org/micks/champmaker/championships/ChampionshipServiceTest.java (hunk 5 above)
Comment (Mickss) [reply src/test/java/org/micks/champmaker/championships/ChampionshipServiceTest.java]: Fixed

**Review by kbnowak — APPROVED** [review]

**Review by tknowak — APPROVED** [review]

**Pull request merged**